packages/blueprints/blueprint/src/ui-selectors/options-schemas.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { Component } from 'projen';
+import { DynamicKVInput } from './dynamic-kv-input';
+import { Blueprint } from '../blueprint';
+
+/**
+ * This represents an options schema based on DynamicKVInput types
+ */
+export type KVSchema = DynamicKVInput[];
+
+/**
+ * This represents all wizard supported schemas.
+ */
+export type OptionsSchemaType = KVSchema;
+
+/**
+ * This represents a section of the wizard that can be dynamically rendered from a schema provided by the synthesis process.
+ */
+export type OptionsSchemaDefinition<
+  // @ts-ignore:next-line value is never read
```

Comment:
If you inline all of this, you can put a single 
```
// @ts-ignore:next-line value is never read
```
Above the type I think

Reply:
There seems to be a bug where JSDoc and TS directives expect the next line to be code.  If JSDoc is first, then it fails because the next line is not code.  If TS directive is first, then the next line is not code.